Makefile

```diff
@@ -96,6 +96,7 @@ server-without-k8s: dep-check
 	go run main.go error.go;
 
 server: dep-check
+	npx husky install
```

Comment:
Will you add `npx` and `husky` to the `dep-check` target?

Reply:
I am using npx so that there is no need to install husky for any contributors you try it out by running `npx husky` command in terminal. And it is mostly likely that every contributor will be installing npm so there will no need of dep-check i guess if that sound good?